packages/ag-charts-community/src/chart/factory/expectedEnterpriseModules.ts

```diff
@@ -26,6 +26,7 @@ export const EXPECTED_ENTERPRISE_MODULES: EnterpriseModuleStub[] = [
         chartTypes: ['cartesian', 'polar', 'hierarchy'],
         identifier: 'gradient',
     },
+    { type: 'root', optionsKey: 'miniChart', chartTypes: ['cartesian'], identifier: 'mini-chart' },
```

Comment:
I'm debating if we should add a `type: 'navigator'` case, as this isn't technically an (options) root module - could we do that relatively easily?